src/mplfinance/plotting.py

```diff
@@ -855,18 +855,18 @@ def plot( data, **kwargs ):
         save = config['savefig']
         if isinstance(save,dict):
             if config['tight_layout'] and 'bbox_inches' not in save:
-                plt.savefig(**save,bbox_inches='tight')
+                fig.savefig(**save,bbox_inches='tight')
             else:
-                plt.savefig(**save)
+                fig.savefig(**save)
         else:
             if config['tight_layout']:
-                plt.savefig(save,bbox_inches='tight')
+                fig.savefig(save,bbox_inches='tight')
             else:
-                plt.savefig(save)
+                fig.savefig(save)
         if config['closefig']: # True or 'auto'
             plt.close(fig)
     elif not config['returnfig']:
-        plt.show(block=config['block']) # https://stackoverflow.com/a/13361748/1639359 
+        fig.show() # https://stackoverflow.com/a/13361748/1639359 
```

Comment:
According to [**the documentation, `Figure.show()`**](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.show) "does not manage an GUI event loop. Consequently, the figure may only be shown briefly or not shown at all if you or your environment are not managing an event loop.  If you're running a pure python shell or executing a non-GUI python script, you should use [matplotlib.pyplot.show](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.show.html#matplotlib.pyplot.show) instead, which takes care of managing the event loop for you."

Therefore I am going to change this back to `plt.show()`.  Otherwise all of the changes look fine and pass unit tests.  Will be merging soon.